test/system/075-exec.bats

```diff
@@ -166,4 +166,19 @@ load helpers
     run_podman rm -f -t0 $cid
 }
 
+@test "podman exec - does not leak session IDs on invalid command" {
+    run_podman run -d $IMAGE top
+    cid="$output"
+
+    for i in {1..3}; do
+        run_podman 127 exec $cid blahblah
+        run_podman 125 exec -d $cid blahblah
+    done
+
+    run_podman inspect $cid
+    inspect_data="$output"
+    run jq '.[].ExecIDs | length' <<< "$inspect_data"
```

Comment:
no need for jq just add `--format "{{.ExecIDs}}"` then you can match the string against `[]`
or if you really want to check by number use `--format "{{len .ExecIDs}}"`

Reply:
good idea, I'll use that